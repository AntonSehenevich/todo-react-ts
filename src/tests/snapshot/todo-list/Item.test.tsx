import renderer from 'react-test-renderer'
import Item from '../../../components/todos/item'

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Item
        handleDragStart={jest.fn()}
        handleDragEnter={jest.fn()}
        handleDragEnd={jest.fn()}
      />
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
