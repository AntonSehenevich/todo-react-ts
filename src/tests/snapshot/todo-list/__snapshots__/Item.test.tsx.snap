// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="item"
  draggable={true}
  onDragEnd={[MockFunction]}
  onDragEnter={[MockFunction]}
  onDragOver={[Function]}
  onDragStart={[MockFunction]}
/>
`;
